bc_obps/registration/api/v2/_operations/_operation_id/_registration/operation_representative.py

```diff
@@ -28,3 +32,20 @@ def create_operation_representative(
     return 200, OperationServiceV2.create_operation_representative(
         get_current_user_guid(request), operation_id, payload
     )
+
+
+@router.put(
+    "/v2/operations/{uuid:operation_id}/registration/operation-representative",
+    response={200: OperationRepresentativeOut, custom_codes_4xx: Message},
+    tags=V2,
+    description="""Removes operation representative from an operation.
+    The endpoint ensures that only authorized industry users can update operations belonging to their operator. Unauthorized access attempts raise an error.""",
+    auth=authorize("approved_industry_user"),
+)
+@handle_http_errors()
+def remove_operation_representative(
+    request: HttpRequest, operation_id: UUID, payload: OperationRepresentativeRemove
+) -> Tuple[Literal[200], None]:
```

Comment:
@andrea-williams , is it fine to return None here? I don't think we actually need anything back. Even though it's None, what I actually get back is {'id': None}, not straight up None or {}